### DISCUSSION TITLE
The error occurred at step 0 8, it seems that there is no job.json file in iter. 000000/02. fp/task */

### DISCUSSION THREAD

**Discussion by XuFanffei** [deepmodeling/dpgen#1381] [Q&A]
The error occurred at step 0 8, and it seems that there is no job.json file in iter. 000000/02. fp/task */. I compared and found that there was a job. json link file in the previous DPgen run, which was linked to 01. model_ devi, it contains information such as ensemble temperature and pressure. But this information is not applicable in the dpgen simplify step, so I don't know how to solve this problem. In addition, the 0th round of dpgen simplify is a random selection of data, and there is no actual use of MD for any model_devi calculation.
on bohrium with image registry.dp.tech/dptech/dpgen:0.10.6

**Comment by njzjz** [comment]
Simplify never runs MD simulations. The behavior looks expected.
